packages/astro/package.json

```diff
@@ -9,30 +9,44 @@
   "engines": {
     "node": ">=18.14.1"
   },
-  "main": "build/cjs/index.server.js",
+  "type": "module",
+  "main": "build/cjs/index.client.js",
   "module": "build/esm/index.server.js",
   "browser": "build/esm/index.client.js",
   "types": "build/types/index.types.d.ts",
+  "exports": {
+    ".": {
+      "node": "./build/esm/index.server.js",
+      "browser": "./build/esm/index.client.js",
+      "import": "./build/esm/index.client.js",
+      "require": "./build/cjs/index.server.js",
+      "types": "./build/types/index.types.d.ts"
+    },
+    "./integration": {
+      "default": "./build/integration/index.ts"
```

Comment:
Hmm I wonder, do we even need to copy this to build, or could we not simply export this from `./src/integration/inedx.ts` and skip the `build:sync:integration` step? 🤔

Reply:
If we didn't over-engineer our build directory structure in v7 it would work 😅 The thing is, right now everything that goes into the tarball is packaged from _within_ `build`. So I somehow need to get it in there. We usually do this in the `prepack` script but for this case, IMO it makes more sense to copy the files at build time rather than at packaging time. Also makes it easier to use the integration when only `yarn link`ing the package.